opportunity/src/app/data/models/ngo.model.ts

```diff
@@ -1,19 +1,35 @@
 import { Opportunity } from './opportunity.model';
 
 export class NGO {
-  id?: string;
   name: string;
   username: string;
   image?: string;
-  about: string;
+  about?: string;
   rating?: number;
-  contact: {
+  contact?: {
     website?: string;
     address: string;
     publicEmail: string;
     phone: string;
   };
   opportunity?: Object;
+  isComplete?: boolean;
+
```

Comment:
this is not good for preventing data validation issues. we should do this validation in the db . 
Otherwise if the form validation fails, we will send null in the db for the required fields